lib/apnagent/message.js

```diff
@@ -139,12 +139,18 @@ Message.prototype.alert = function (key, value) {
   var allowed = [
       'body'
     , 'action-loc-key'
+    , 'actions'
     , 'loc-key'
     , 'loc-args'
     , 'launch-image'
     , 'title'
     , 'title-loc-key'
     , 'title-loc-args'
+ ],
+      valueMustBeArray = [
```

Comment:
style ocd: of your `var` declaration is more than one line then use the `var` keyword for each declaration....

```js
var arr1 = [
  // ..
]

var arr2 = [
  // ...
]
```